jenkins_jobs/trigger_rpmlint_from_versions_repo.groovy

```diff
@@ -1,19 +1,20 @@
-job('trigger_rpmlint_from_builds_repo') {
+job('trigger_rpmlint_from_versions_repo') {
   label('validation_slave_label')
   concurrentBuild()
   parameters {
-    stringParam('sha1', '', 'SHA-1 of the commit to validate.')
+    stringParam('builds_sha1', 'master', 'SHA-1 of the builds commit')
```

Comment:
Is this variable here so that we can run the job manually? Because I don't think the trigger plugin has a way of setting it, right?